src/node/node_state.h

```diff
@@ -900,7 +900,7 @@ namespace ccf
             // Inform snapshotter of all signature entries so that this node can
             // continue generating snapshots at the correct interval once the
             // recovery is complete
-            snapshotter->record_committable(last_recovered_idx);
+            snapshotter->record_committable_unsafe(last_recovered_idx);
             snapshotter->commit(last_recovered_idx, false);
```

Comment:
I believe this path was to maintain the ledger file size invariant during the recovery, which I think can now go?

Reply:
Yep, I think so too. Just waiting for the CI to confirm that this doesn't break anything.